wrapanapi/lenovo.py

```diff
@@ -307,3 +310,17 @@ def set_server_led_blinking(self, server_name):
         response = self.change_led_status(server, led['name'], 'Blinking')
 
         return "LED state action has been sent, status:" + str(response.status_code)
+
+    def stats(self, *requested_stats, **kwargs):
+        requester = None
+
+        # Get the requester which represents the class of this method's caller
+        if 'requester' in kwargs:
+            requester = kwargs.get('requester')
+
+        # Retrieve and return the stats
+        requested_stats = requested_stats or self._stats_available
+        return {stat: self._stats_available[stat](self, requester) for stat in requested_stats}
+
+    def disconnect(self):
+        pass
```

Comment:
Please add a logging statement here if there's no disconnect action to be taken.